### ISSUE TITLE
Регистрация

### ISSUE THREAD

**Issue by Lanaken** [bmstu-iu9/utp2018-7-forum#6]
Когда я попытался зарегистрировать одного и того же пользователя 2-ой раз, у меня открылась пустая страничка с надписью "something went wrong".

**Comment by Lanaken** [comment]
Ожидаемое поведение: при попытке зарегистрировать существующего пользователя должна вылезать плашка с надписью о том, что таковой уже существует.

**Comment by Lanaken** [comment]
Использовался Google Chrome

**Comment by Lanaken** [comment]
Та же самая проблема, если окна заполняются кириллицей.
Т.е. вместо плашки о некорректности введенных данных открывается пустое окно с надписью "something went wrong".

**Comment by SpeedyGonzaless** [comment]
пофикшено с кириллицей

**Comment by Lanaken** [comment]
Да,осталась только проблема при регистрации существующего пользователя.
И еще вопрос, при вводе неправильного пароля открывается пустая страница с надписью "password mistmatch".Должна ли открываться новая страница?
